scss/civicrm/common/_tables.scss

```diff
@@ -333,3 +332,20 @@ table {
     @extend %civihr-table;
   }
 }
+
+.dataTables_wrapper {
```

Comment:
can you explain here what this generic style is for? so we can figure out how to document it

Reply:
This is for data tables, any data table in CiviCRM should have padding as defined in here.
CiviCRM does have tables for form as well. so, we are targeting only tables which are datatable using this wrapper and giving accurate padding to it.

Reply:
do you think a `_datatables.scss` partial would make sense? what's your opinion?

Reply:
Yeah, that makes sense. 
I'm also planning to move other style of data tables to that file when it comes to any related ticket so, we can have it on one file currently it is having it defined on different pages.
Though, I'm not doing all things in one step so, things stay more clear when we do it.